code/addons/a11y/src/components/A11yContext.tsx

```diff
@@ -317,8 +328,9 @@ export const A11yContextProvider: FC<PropsWithChildren> = (props) => {
 
     const others = results?.[tab as RuleType]
       .flatMap((r) => r.nodes.flatMap((n) => n.target).map(String))
-      .filter((e) => !selected.includes(e));
+      .filter((e) => !['html', 'body', 'main', ...selected].includes(e));
     emit(HIGHLIGHT, {
+      id: `${ADDON_ID}/others`,
       selectors: others,
       styles: {
         outline: `1px solid color-mix(in srgb, ${colorsByType[tab]}, transparent 30%)`,
```

Comment:
I'm wondering why we want to allow custom styling.

The addon probably already did support it, so we might just be keeping it for the sake of not changing, but I do want to raise the question if it's fundamentally a good idea.

For UI consistency reasons we might not want this.. @MichaelArestad 

But also if these styles are going to be directly injected.. that might raise security concerns? @shilman

Reply:
I'm not sure what your security concerns are here. A malicious addon can already do pretty bad things without going through a highlight...

Reply:
Some folks might want custom styling if our highlight colors are close to their brand colors. I don't think it's a bad idea. I also don't think many folks will opt to customize it.